### DISCUSSION TITLE
denerf certains items ?

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1181] [Suggestions fermées]
fortresse japonnaise => epique avec 168 de shield
bouclier de maitre => legendaire avec 169 de shield

Y'a un problème quelque part

**Comment by BastLast** [comment]
Il n'y a pas de soucis du tout,

La rareté n'est pas forcément gage de meilleure qualité :)

**Comment by ghost** [comment]
la logique de mettre des raretés du coup ?

**Reply by BastLast** [reply]
Que certains items apparaissent moins d'autres

**Reply by BastLast** [reply]
Que certains items apparaissent moins d'autres

**Comment by ghost** [comment]
oui mais genre à quoi ca sert de faire des items qui apparaissent moins que les autres si ils sont moins utiles ...

**Reply by BastLast** [reply]
Ca apporte de la diversité

**Comment by ghost** [comment]
les items c'est pas cosmetique et facultatif si jamais ... ca sert en fight par exemple
la logique semble m'échapper pour le coup : à quoi ca sert de proposer des items "plus rares" si ils sont moins bon que les items facilement trouvables ? le but d'avoir des items plus difficiles à avoir, c'est qu'ils soient meilleurs, c'est la base de tout rpg en fait donc à part faire différamment pour faire différamment je vois pas ... surtout que maintenant y'a le mission shop aussi qui permet d'avoir ses items "plus rares" mais inutiles ... 30 diamants pour ca ...

**Reply by QQtin3** [reply]
Étant le créateur du bot, il me semble évident qu'il sache l'utilité des items. Cependant les items légendaires sont un peu mieux que les épiques, tu n'as pas eu de chance et tu es tombé sur le pire item légendaire de défense, les autres items sont un peu meilleurs.
Puis le prix de vente de l'objet en question n'est pas le même, et comme dit plus haut, c'est pas parce que c'est plus rare que ça doit être meilleur, même si c'est le cas sur le bot (sauf cas exceptionnel) c'est le cas sur les items légendaire et mythique par exemple. Les mythiques sont plus rare mais pour autant pas plus fort. EDIT: Les mythiques sont équivalent aux légendaires au niveau des stats. Mais ils sont pas forcément plus puissant, certains le sont

**Reply by BastLast** [reply]
Totalement d'accord avec tout ce qu'a dit quenquentin, je ne vois pas le problème à ce qu'il y ai un côté cosmétiques aux items, je pense que dans la globalité l'ordre est respecté mais je ne vois pas le soucis à ce qu'il y ai des exceptions x)
